db/controllers/data.controller.js

```diff
@@ -0,0 +1,10 @@
+const pool = require('../connection.js')
```

Comment:
Do you need to require your connection pool into the controller? Are you using it anywhere? 🤔